### PR TITLE
[TASK] add missing swedish translation for labels

### PR DESCRIPTION
**Pull request by mesosa**

### CODE REVIEW

File: Resources/Private/Language/sv.locallang.xlf [hunk 1]
```diff
@@ -4,6 +4,12 @@
           product-name="pxa_product_manager" target-language="sv">
         <header/>
         <body>
+            <trans-unit id="fe.go_back">
+                <source>Tillbaka</source>
+            </trans-unit>
+            <trans-unit id="fe.single_view_disabled">
+                <source>Enkel visning är avstängd för den här artikeln!</source>
```
Comment (MattiasNilsson) [inline Resources/Private/Language/sv.locallang.xlf]: Detaljvisning instead of Enkel visning.

File: Resources/Private/Language/sv.locallang.xlf [hunk 2]
```diff
@@ -16,18 +22,184 @@
             <trans-unit id="fe.attributes">
                 <target>Produktdata</target>
             </trans-unit>
+            <trans-unit id="fe.latest_visited_products">
+                <source>Senaste besökta produkterna</source>
```
Comment (MattiasNilsson) [inline Resources/Private/Language/sv.locallang.xlf]: Senast

File: Resources/Private/Language/sv.locallang.xlf [hunk 3]
```diff
@@ -16,18 +22,184 @@
...
+                <source>Inga produkter hittade</source>
+            </trans-unit>
+            <trans-unit id="fe.load_more">
+                <source>Ladda in fler</source>
+            </trans-unit>
+            <trans-unit id="fe.not_found_header">
+                <source>Hittades inte</source>
+            </trans-unit>
+            <trans-unit id="fe.not_found_text">
+                <source>Porudkten hittades inte</source>
```
Comment (MattiasNilsson) [inline Resources/Private/Language/sv.locallang.xlf]: Produkten

File: Resources/Private/Language/sv.locallang.xlf (hunk 1 above)
Comment (mesosa) [reply Resources/Private/Language/sv.locallang.xlf]: Changed

File: Resources/Private/Language/sv.locallang.xlf (hunk 2 above)
Comment (mesosa) [reply Resources/Private/Language/sv.locallang.xlf]: Changed

File: Resources/Private/Language/sv.locallang.xlf (hunk 3 above)
Comment (mesosa) [reply Resources/Private/Language/sv.locallang.xlf]: Changed

**Comment by MattiasNilsson** [comment]
Nice! 👍

**Pull request merged**